<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLineEdit" name="IPLine">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>192.168.56.1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PortLine">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>69</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelIP">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Server IP:</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelPort">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Server Port:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GetBtn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Get</string>
   </property>
  </widget>
  <widget class="QListView" name="MessageList">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>741</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SendBtn">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>130</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ShowDirBtn">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Dir</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="FileLine">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LabelFile">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>File:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
